README.md

```diff
@@ -711,3 +713,33 @@ cast send <SOURCE_MINTER_ADDRESS> --rpc-url avalancheFuji --private-key=$PRIVATE
 ```
 
 depending on whether you filled the [`SourceMinter.sol`](./src/cross-chain-nft-minter/SourceMinter.sol) contract with `Native (0)` or `LINK (1)` in step number 3.
+
+## CCT Standard
+
+The [Cross-Chain Token (CCT)](https://docs.chain.link/ccip/concepts/cross-chain-token/overview) standard offers a streamlined and decentralized approach to enabling token transfers across blockchains using Chainlink's Cross-Chain Interoperability Protocol (CCIP).
```

Comment:
Suggest a more direct explanation for CCT as follows (or similar):

******
The [Cross-Chain Token (CCT)](https://docs.chain.link/ccip/concepts/cross-chain-token/overview) standard provides a standard for transferring tokens between different blockchains using Chainlink’s Cross-Chain Interoperability Protocol (CCIP).

By implementing the CCT standard, you can enable seamless cross-chain transfers for your own "custom" ERC20 tokens through CCIP.
******